config/locales/en.yml

```
@@ -1486,6 +1486,13 @@ risc.security_event.errors.typ_error: typ header must be %{expected_typ}
 saml_idp.auth.error.title: Error
 saml_idp.shared.saml_post_binding.heading: Submit to continue
 saml_idp.shared.saml_post_binding.no_js: JavaScript seems to be turned off in your browser. Normally this step happens automatically, but because you have JavaScript turned off, please click the submit button to continue signing in or signing out.
+security_check_failed.contact: Contact %{app_name} support
+security_check_failed.details: We don’t recognize the device or browser you’re signing in from.
+security_check_failed.info_p1: 'Use the same device and browser you created your %{app_name} account with.'
+security_check_failed.info_p2_html: Try <a href="%{piv_cac_url}">signing in with your government employee ID</a> if you are a government employee.
```

IMO I think it might be a little more reliable to get things through the i18n process to break the link title into its own key
```suggestion
security_check_failed.info_p2_html: Try %{piv_cac_link_html} if you are a government employee.
security_check_failed.info_p2_link_html: signing in with your government employee ID
```

and then update the callsite to be like
```erb
    <li><%= t('security_check_failed.info_p2_html', piv_cac_link_html: link_to(t('security_check_failed.info_p2_link_html'), login_piv_cac_url) %></li> 
```